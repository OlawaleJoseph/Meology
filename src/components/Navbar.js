import React from 'react';
import { Link } from 'react-router-dom';
import logo from '../logo.png';

export default function Navbar() {
  return (
    <header className="navbar">
      <nav className="nav-center">
        <Link to="/">
          <img src={logo} alt="Meology logo" className="logo" />
        </Link>
        <ul className="nav-links">
          <li>
            <Link to="/">home</Link>
          </li>
          <li>
            <Link to="/about">about</Link>
          </li>
        </ul>
      </nav>
    </header>
  );
}
