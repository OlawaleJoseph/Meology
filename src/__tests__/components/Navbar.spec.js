import React from 'react';
import renderer from 'react-test-renderer';
import { Router } from 'react-router-dom';
import { createMemoryHistory } from 'history';
import Navbar from '../../components/Navbar';

describe('Home Page Snapshot', () => {
  it('builds home page correctly with search and meallist', () => {
    const tree = renderer.create(
      <Router history={createMemoryHistory()}>
        <Navbar />
      </Router>,
    );
    expect(tree).toMatchSnapshot();
  });
});
