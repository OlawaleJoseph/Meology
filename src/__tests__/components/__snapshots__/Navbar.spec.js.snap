// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Page Snapshot builds home page correctly with search and meallist 1`] = `
<header
  className="navbar"
>
  <nav
    className="nav-center"
  >
    <a
      href="/"
      onClick={[Function]}
    >
      <img
        alt="Meology logo"
        className="logo"
        src="logo.png"
      />
    </a>
    <ul
      className="nav-links"
    >
      <li>
        <a
          href="/"
          onClick={[Function]}
        >
          home
        </a>
      </li>
      <li>
        <a
          href="/about"
          onClick={[Function]}
        >
          about
        </a>
      </li>
    </ul>
  </nav>
</header>
`;
